tests/pytest.ini

```diff
@@ -0,0 +1,4 @@
+# Disable stout capture, as it conflicts with Fabric
+#  see https://github.com/pytest-dev/pytest/issues/1585
+[pytest]
+addopts = --capture=no
```

Comment:
I didn't know if it is a problem specific to Fabric 2 or not. I can add it to other repos next time I submit a PR there.

Reply:
One question, though.

As you have seen, I added pytest.ini both to the root directory and to tests, so that you could potentially run pytest from both of them. The error message you when when running pytest without `-s` is quite hard to understand, so I preferred to make sure we don't run into the problem.

What do you think about this?

Reply:
Hmm, I think that pytest will collect (and run) Python code in all sub directories of the folder where you start it, so it may have unintended side effects if we advertise that it can be run outside the test folder. I'm not strictly opposed, but I think my vote would be to keep this in the test directory and always run it from there.

Reply:
I agree.

I will let the test framework fail if you run it from the root repo directory.

Reply:
Oh! I just realized that I actually didn't commit the extra pytest.ini, so it was only on my local copy. Nothing to do there, then.